pkg/rekor/identity.go

```diff
@@ -81,6 +82,9 @@ type MonitoredValues struct {
 	// OIDMatchers contains a list of OID extension fields and associated values
 	// ex. Build Signer URI, associated with specific workflow URIs
 	OIDMatchers []OIDMatcher `yaml:"oidMatchers"`
+	// FulcioExtensions contains all OID extensions currently supported by Fulcio
+	// each extension has a list of values to match on, ex. `build-signer-uri`
+	FulcioExtensions extensions.FulcioExtensions `yaml:"fulcioOIDExtensions"`
```

Comment:
Can we rename the yaml tag to be `fulcioExtensions`, and update the documentation? We don't need to include OIDs, as that is only an identifier for an extension.